pages/help/help_account-management.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Account management
+permalink: /help/account-management/
+layout: layouts/info-page
+searchbox: true
```

Comment:
Suggestion: It's not doing any harm, but I think let's take out `search box` from this page since it's never an option for the `info-page` type. Wondering if leaving it in may cause confusion in the future. 

Leaving the comment on this one, but this will apply to all of them